instagram/src/App.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import PostContainer from './components/PostContainer';
+import './App.css';
+import dummyData from './dummy-data';
+
+class App extends Component {
+  render() {
+    return (
+      <div>
+        <h1>instagram</h1>
+        <span>
+          {dummyData.map((post,i)=><div key={i}><PostContainer container={post}/></div>)}
+        </span>
+        <div><PostContainer/></div>
```

Comment:
Since you are mapping over dummy data and filling in your posts that way, you probably don't need this, as it's going to be blank